chia-tools/graph.gnuplot

```diff
@@ -67,3 +72,15 @@ set xrange [0:2000]
 plot "chain-resource-usage-cdf.dat" using 11:1 with lines title "block decompress + parse", \
 "chain-resource-usage-cdf.dat" using 12:1 with lines title "block reference lookup", \
 "chain-resource-usage-cdf.dat" using 14:1 with lines title "conditions"
+
+set output "block-wall-clock-delta-cdf.png"
```

Comment:
Whats the meaning of the `cdf` postfix used in this file?

Reply:
cumulative distribution function. The plot is essentially the integral of the frequency function of the samples